MessageViewController/MessageAutocompleteController.swift

```diff
@@ -29,6 +29,27 @@ public final class MessageAutocompleteController: MessageTextViewListener {
         public let range: NSRange
     }
     public private(set) var selection: Selection?
+    
+    /// Adds an additional space after the autocompleted text when true. Default value is `TRUE`
```

Comment:
Reminder for me: I should start adding docs 😝

Reply:
😝 Every devs favorite task